chemprop/train/train.py

```diff
@@ -45,67 +47,133 @@ def train(model: MoleculeModel,
+            if args.loss_function == "dirichlet":
+                loss = (
+                    loss_func(preds, targets, args.evidential_regularization)
+                    * target_weights
+                    * data_weights
+                    * mask
+                )
             else:
                 target_losses = []
                 for target_index in range(preds.size(1)):
-                    target_loss = loss_func(preds[:, target_index, :], targets[:, target_index]).unsqueeze(1)
+                    target_loss = loss_func(
+                        preds[:, target_index, :], targets[:, target_index]
+                    ).unsqueeze(1)
                     target_losses.append(target_loss)
-                loss = torch.cat(target_losses, dim=1).to(torch_device) * target_weights * data_weights * mask
-        elif args.dataset_type == 'spectra':
+                loss = (
+                    torch.cat(target_losses, dim=1).to(torch_device)
+                    * target_weights
+                    * data_weights
+                    * mask
+                )
+        elif args.dataset_type == "spectra":
             loss = loss_func(preds, targets, mask) * target_weights * data_weights * mask
-        elif args.loss_function == 'bounded_mse':
-            loss = loss_func(preds, targets, lt_target_batch, gt_target_batch) * target_weights * data_weights * mask
-        elif args.loss_function == 'evidential':
-            loss = loss_func(preds, targets, args.evidential_regularization) * target_weights * data_weights * mask
-        elif args.loss_function == 'dirichlet': # classification
-            loss = loss_func(preds, targets, args.evidential_regularization) * target_weights * data_weights * mask
+        elif args.loss_function == "bounded_mse":
+            loss = (
+                loss_func(preds, targets, lt_target_batch, gt_target_batch)
+                * target_weights
+                * data_weights
+                * mask
+            )
+        elif args.loss_function == "evidential":
+            loss = (
+                loss_func(preds, targets, args.evidential_regularization)
+                * target_weights
+                * data_weights
+                * mask
+            )
+        elif args.loss_function == "dirichlet":  # classification
+            loss = (
+                loss_func(preds, targets, args.evidential_regularization)
+                * target_weights
+                * data_weights
+                * mask
+            )
+        elif args.loss_function == "quantile":
+            loss = loss_func(preds, targets, args.quantile) * target_weights * data_weights * mask
+        elif args.loss_function == "quantile_interval":
+            quantiles_tensor = torch.Tensor(args.quantiles).to(torch_device)
+            loss = (
+                loss_func(preds, targets, quantiles_tensor)
+                * target_weights
+                * data_weights
+                * mask
+            )
```

Comment:
```suggestion
        elif args.loss_function == "quantile_interval":
            quantiles_tensor = torch.tensor(args.quantiles, device=torch_device)
            loss = (
                loss_func(preds, targets, quantiles_tensor)
                * target_weights
                * data_weights
                * mask
            )
```

Reply:
This removes the mention of the quantile loss function and adds the device inside the initial tensor creation.